app/_locales/en/messages.json

```diff
@@ -448,6 +463,24 @@
   "encryptNewDen": {
     "message": "Encrypt your new DEN"
   },
+  "endOfFlowMessage1": {
+    "message": "You passed the test - keep your seedphrase safe, it's your responsibility!"
+  },
+  "endOfFlowMessage2": {
+    "message": "Tips on storing it safely "
+  },
+  "endOfFlowMessage3": {
+    "message": "• Save a backup in multiple places"
+  },
+  "endOfFlowMessage4": {
+    "message": "• Never tell anyone"
+  },
+  "endOfFlowMessage5": {
+    "message": "If you need to back it up again, you can find them in Settings -> Security."
```

Comment:
Nit: `s/find them/find it/` I think? Or maybe `s/back it up/back them up/` if that works?